tests/publiccloud/ssh_interactive_start.pm

```diff
@@ -16,17 +16,39 @@ use Mojo::Base 'publiccloud::ssh_interactive_init';
 use publiccloud::ssh_interactive;
 use testapi;
 use utils;
+use publiccloud::utils "select_host_console";
 
 sub run {
     my ($self, $args) = @_;
-    select_console 'tunnel-console';
+
+    # This ensure that we have used the setup console, even if no module was run before.
+    select_host_console();
+    my $setup_console = current_console();
 
     # Establish the tunnel (it will stay active in foreground and occupy this console!)
+    select_console('tunnel-console');
     ssh_interactive_tunnel($args->{my_instance});
 
-    # Switch to root-console and SSH to the instance
-    # every other loaded test must stay in root-console
-    select_console 'root-console';
+    # Fix serial terminal to use TUNNELED stuff
```

Comment:
Can you rephrase that comment to avoid "Fix" and "stuff" and say what the real issue is? :)